matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/summary/RoomSummaryDataSource.kt

```diff
@@ -193,7 +194,7 @@ internal class RoomSummaryDataSource @Inject constructor(
         }
         val dataSourceFactory = realmDataSourceFactory.map {
             roomSummaryMapper.map(it)
-        }
+        }.map { it.getWithParents() }
```

Comment:
I think it would be cool to have an overload of `map` in `RoomSummaryMapper` to have a param to request flattened parent

Reply:
I think it would be better also. Like that it won't impact perf for those who don't need to load additional things?

Reply:
Good shout!